src/components/tabs/TabNav.js

```diff
@@ -0,0 +1,25 @@
+export const TabNav = ({ tabs, selectedIndex, selectTab }) => {
+  const day = ['ПН', 'ВТ', 'СР', 'ЧТ', 'ПТ', 'СБ', 'ВС'];
+
+  return (
+    <div className="nav-tab">
+      <div className="lists">
+        {
+          tabs.map((task, index) => (
+            <div
+              key={index}
+              className={(selectedIndex === index) ? 'active' : ''}
+            >
+              <a
+                href="#"
+                onClick={event => selectTab(event, index)}
+              >
+                {day[index]}
```

Comment:
TabNav shouldn't know any information about days. It renders only text which provided to it via props
```
<a
    href="#"
    onClick={event => selectTab(event, index)}
>
    {tab}
</a>
```